tests/test_dockerprovisioner.py

```diff
@@ -20,7 +20,8 @@
 
 import testtools
 
-import molecule.provisioners as provisioners
+from molecule.Provisioners import DockerProvisioner
```

Comment:
Since we created a Provisioners directory, can we do the same with the tests to follow the same structure as the code?